autopts/wid/cap.py

```diff
@@ -772,6 +772,14 @@ def hdl_wid_402(_: WIDParams):
     return True
 
 
+def hdl_wid_404(_: WIDParams):
+    """
+        Please wait to establish 2 connections. After that, Click OK to send notification.
```

Comment:
no need to actually wait for those connections here?

Reply:
Before this wid, all PTS are connected via the serialized auto-pts sync functions.

Even with the sync functions, this means that the dialog might be OK'ed before the second device is connected. 
It didn't seem to make a difference.

If I read the test spec correctly, some of the CAP/COM/CRC tests expects that a notification from one PTS triggers and update on the other PTS via the IUT, but PTS does not test for that. The new wids don't cheat (e.g. implementing this functionality manually in auto-pts), so I think it's good enough.